client/my-sites/exporter/export-card/index.jsx

```diff
@@ -91,15 +94,16 @@ const mapStateToProps = ( state, { siteId } ) => ( {
 
 const trackExportClick = ( scope = 'all' ) => recordTracksEvent( 'calypso_export_start_button_click', { scope } );
 
-const mapDispatchToProps = ( dispatch, { siteId } ) => ( {
+const mapDispatchToProps = ( dispatch, ownProps ) => ( {
```

Comment:
Any reason why you dislike the [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here? I personally find that it saves a bit of typing, for example:
```javascript
const mapDispatchToProps = ( dispatch, { siteId, siteType } ) => ( {
//...
     fetchStatus: () => dispatch( exportStatusFetch( siteId ) ),
     exportAll: () => dispatch( withAnalytics( trackExportClick(), startExport( siteId, { exportAll: true }, siteType )
//...
```